_rules/iframe-not-focusable-has-no-interactive-content-akn7bn.md

```diff
@@ -28,14 +28,19 @@ acknowledgments:
 
 ## Applicability
 
-This rule applies to any non-focusable `iframe` element that has [focusable][] content.
+This rule applies to any `iframe` element that [contains](#akn7bn:contain) at least one element for which all the following are true:
 
-## Expectation
-
-For each test target, the [nested browsing context][] does not [contain](#akn7bn:contain) elements that are [visible][] and part of the [sequential focus navigation][].
+- the element is [visible][]; and
+- the element is part of the [flattened tabindex-ordered focus navigation scope][] of the `iframe`.
 
 An element is <dfn id="akn7bn:contain">contained</dfn> in a [nested browsing context][] if its [owner document][] is the [container document][] of the [nested browsing context][].
 
+The [flattened tabindex-ordered focus navigation scope][] of an `iframe` is essentially the [sequential focus navigation][] order inside its content.
```

Comment:
You don't need this. All documents have sequential focus navigation, an iframe has a document, ergo. Iframes can have sequential focus navigation.

Reply:
Indeed 🙈 
I somehow got lost in technicalities and forgot that the sequential focus navigation is indeed exactly the FTOFNS 🙃 
(well, not exactly, because the former is an order and the later is a list 🤓, but close enough)